### PR TITLE
Ability to sort generators in custom order

### PR DESCRIPTION
**Pull request by agmitron**

### CODE REVIEW

File: packages/jmix-front-generator/src/common/base-generator.ts [hunk 1]
```diff
@@ -185,6 +185,7 @@ export interface GeneratorExports {
   params?: StudioTemplateProperty[],
   description?: string;
   icon?: string;
+  index: number;
```
Comment (web-devel) [inline packages/jmix-front-generator/src/common/base-generator.ts:188]: Let's make it options

File: packages/jmix-front-generator/package-lock.json [hunk 2]
```diff
@@ -1,78 +1,27 @@
 {
 	"name": "@haulmont/jmix-front-generator",
 	"version": "1.0.0-next.17",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```
Comment (web-devel) [inline packages/jmix-front-generator/package-lock.json:4]: Please use npm 7 (it creates lock files v2)

File: packages/jmix-front-generator/src/init.ts [hunk 3]
```diff
@@ -81,15 +82,18 @@ function collectGenerators(generatorsDir: string, genFileName?: string): Generat
...
       const params = generatorExports.params;
       const description = generatorExports.description;
+      const index = generatorExports.index;
 
       const iconPath = generatorExports.icon
         ? path.relative(process.cwd(),path.join(generatorPath, generatorExports.icon))
         : undefined;
 
-      generators.push({name, options, params, description, iconPath});
+      // generators.push({name, options, params, description, iconPath, index});
```
Comment (web-devel) [inline packages/jmix-front-generator/src/init.ts:91]: Don't leave commented code (unless there is an exceptional reason for that)

**Review by web-devel — CHANGES_REQUESTED** [review]

**Review by web-devel — CHANGES_REQUESTED** [review]
Please fix the commit message (use `npm run commit`)
* it's feature, not refactor
* add ticket number
* add `affects` label

**Comment by agmitron** [comment]
> Please fix the commit message (use `npm run commit`)
> 
> * it's feature, not refactor
> * add ticket number
> * add `affects` label

Resolved.

**Review by vyacheslav-pushkin — APPROVED** [review]

**Pull request merged**